jetstream/experimenter.py

```diff
@@ -48,7 +48,6 @@ class Experiment:
     normandy_slug: Optional[str]
     type: str
     status: Optional[str]
-    active: bool
```

Comment:
What's the reason for removing this?

Reply:
Ah, I missed the last comment. Based on the [spec](https://github.com/mozilla/nimbus-shared/blob/fdb57e96fe7a62d3cdf4666af3e7cdbfccd329fd/types/experiments.ts#L44) end_date can be null...

Reply:
It seems redundant with `status` so I figured we may as well pick the more expressive one.